src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -168,15 +172,20 @@ class SurrealPersistenceService implements PersistenceService {
 
     @Override
     void saveContainerRequest(String token, WaveContainerRecord data) {
-        surrealDb.insertContainerRequestAsync(authorization, token, data).subscribe({ result->
-            log.trace "Container request with token '$token' saved record: ${result}"
-        }, {error->
-            def msg = error.message
-            if( error instanceof HttpClientResponseException ){
-                msg += ":\n $error.response.body"
-            }
-            log.error("Error saving container request record ${msg}\n${data}", error)
-        })
+        final query = "INSERT INTO wave_request ${JacksonHelper.toJsonAndAppend(data, Map.of('id',token))}"
```

Comment:
because previously in /key endpoint, we were passing token as id:
https://github.com/seqeralabs/wave/blob/9ace76e7d6a65d40c9277418b2d000c9c4897e52/src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy#L171

and we use it when we update container request, so to make it backward compatible and use token to update container request, we need to add it as an id in sql

Reply:
https://github.com/seqeralabs/wave/blob/9ace76e7d6a65d40c9277418b2d000c9c4897e52/src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy#L183-L187

Reply:
if we do not provide id, then surrealdb will auto generate like it does in `wave_build` table
But there we use buildId to traverse the table

Reply:
but i think its better ti have Id to take advantage on indexes, in case of wave_build when data will grow the queries will start to slow down, because buildId is not indexed

Reply:
Understand, then it's better the approach of having `id` in the `WaveContainerRecord` adding a comment explaining that's the token and it should be named `id` as surreal requirement. 

Also it would still useful to add a validation check the prevent conda file (and docker file) bigger than 100k 

likely here https://github.com/seqeralabs/wave/blob/943f52eac3d0094f5d8b881841a2ed6a9d928f8a/src/main/groovy/io/seqera/wave/controller/ContainerController.groovy#L524-L533

Reply:
ok understood, i will make the chnages

Reply:
done